Source/Core/Core/IOS/FS/HostBackend/FS.cpp

```diff
@@ -254,12 +254,12 @@ HostFileSystem::FstEntry* HostFileSystem::GetFstEntryForPath(const std::string&
 
 void HostFileSystem::DoState(PointerWrap& p)
 {
-  // Temporarily close the file, to prevent any issues with the savestating of /tmp
+  // Temporarily close the file, to prevent any issues with the savestating of files/folders.
   for (Handle& handle : m_handles)
     handle.host_file.reset();
 
-  // handle /tmp
-  std::string Path = BuildFilename("/tmp").host_path;
+  // Handle saving/loading all files/folders in the NAND recursively (the root folder is "Dolphin Emulator/WiiSession" when a movie is being created/played back, and is "Dolphin Emulator/Wii" otherwise).
+  std::string Path = BuildFilename("/").host_path;
```

Comment:
```suggestion
  std::string Path = BuildFilename(Core::WantsDeterminism() ? "/" : "/tmp").host_path;
```
This could be done to only do a full state when recording a movie (where this would be most desired and least problematic).

Reply:
While the "handle /tmp" comment should be changed, I don't think your suggested replacement is good. The user directory is only called "Dolphin Emulator" on Windows, and even then, only if you're not using portable.txt. I also think listing the paths at all is overly much detail (though this is more subjective, and I'm willing to be overruled).

Reply:
When the user is playing/recording a movie, the path that "/" corresponds to is "Dolphin Emulator/WiiSession". Otherwise, the path that "/" corresponds to is "Dolphin Emulator/Wii".